server/controllers/court-information-two-controller.ts

```diff
@@ -33,8 +31,9 @@ const courtInformationTwoController = async (
 
     const viewModel = mapCaseDetailsToHearingSummary(caseDetails)
     res.locals.hearingData = viewModel
+    res.locals.caseDetails = caseDetails
```

Comment:
Is this relevant for `passport-integration`? If not I would remove it and add a `card` to the backlog for this change.